src/main/java/lotto/model/LottoResultCalculator.java

```diff
@@ -0,0 +1,11 @@
+package lotto.model;
+
+public class LottoResultCalculator {
+    public static LottoResults calculate(Lottos lottos, WinningLotto winningLotto) {
+        LottoResults lottoResults = new LottoResults();
+        for (Lotto lotto : lottos.lottos()) {
+            lottoResults.update(winningLotto.calculatePrize(lotto));
+        }
+        return lottoResults;
```

Comment:
굳이 이 객체가 필요한가?


```suggestion
        LottoResults lottoResults = lottos.match(winningLotto);
```
이와 같이 Lottos에 메시지를 보내 위와 같이 구현해도 되지 않을까?